src/main/java/org/sonarlint/intellij/core/SonarLintProjectNotifications.java

```diff
@@ -94,36 +98,61 @@ public void notifyServerNotUpdated() {
     if (shown) {
       return;
     }
-    Notification notification = new Notification(BINDING_PROBLEM,
-      "SonarLint - No data for SonarQube server",
-      UPDATE_SERVER,
-      NotificationType.ERROR, new OpenGeneralSettingsNotificationListener(myProject));
+    Notification notification = new Notification(GROUP_BINDING_PROBLEM,
+      "<b>SonarLint - No data for SonarQube server</b>",
+        UPDATE_SERVER_MSG,
+      NotificationType.WARNING, new OpenGeneralSettingsNotificationListener(myProject));
     notification.setImportant(true);
     notification.notify(myProject);
     shown = true;
   }
 
-  public void notifyServerNeedsUpdate(String serverId) {
+  public void notifyServerStorageNeedsUpdate(String serverId) {
     if (shown) {
       return;
     }
-    Notification notification = new Notification(BINDING_PROBLEM,
-      "SonarLint - binding for SonarQube server '" + serverId + "' needs to be updated",
-      UPDATE_SERVER,
-      NotificationType.ERROR, new OpenGeneralSettingsNotificationListener(myProject));
+    Notification notification = new Notification(GROUP_BINDING_PROBLEM,
+      "<b>SonarLint - Binding for server '" + serverId + "' outdated</b>",
+        UPDATE_SERVER_MSG,
+      NotificationType.WARNING, new OpenGeneralSettingsNotificationListener(myProject));
     notification.setImportant(true);
     notification.notify(myProject);
     shown = true;
   }
 
+  public void notifyServerHasUpdates(String serverId, List<String> changelog, ConnectedSonarLintEngine engine, SonarQubeServer server, boolean onlyProjects) {
+    Notification notification = new Notification(GROUP_UPDATE_NOTIFICATION,
```

Comment:
What do you think about using the "shown" flag? If a notification is shown the need to update the global storage, I don't think it's useful to show another about available updates.